examples/regtest/cp2k/h2o-mol-nvt-cl/GTH_BASIS_SETS

```diff
@@ -0,0 +1,2139 @@
+H SZV-GTH
```

Comment:
I really do not think it is needed to add the whole GTH basis set file to this. Can you clear it? Please keep only the ones you need for this calculation or find a way to refer it to some other place.

Reply:
I can perhaps change this.

Reply:
I have made the changes.